libvast/CMakeLists.txt

```diff
@@ -393,7 +393,11 @@ find_package(yaml-cpp 0.6.2 REQUIRED)
 target_link_libraries(libvast PRIVATE yaml-cpp)
 dependency_summary("yaml-cpp" yaml-cpp "Dependencies")
 
-# Link against xxhash.
+# Link against xxHash.
+target_compile_definitions(
```

Comment:
Please add a comment so we don't forget to remove this when we update the minimum version of XXHash.